### PR TITLE
Enable tobiko execution on the test-operator zuul job

### PR DESCRIPTION
**Pull request by eduolivares**
Both Tempest and Tobiko will be executed with test-operator.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1171

### CODE REVIEW

**Comment by eduolivares** [comment]
recheck

**Review by lpiwowar — APPROVED** [review]
LGTM!

I'm going to take a look a the mounting of the ssh key so that this is not needed:
```
cifmw_test_operator_tobiko_override_conf:
                rhosp:
                  ssh_key_filename: '~/tobiko_keys/osp_ssh_key'
```
In the meantime I believe we can merge this once the ci-framework change gets merged.

**Review by cjeanner — APPROVED** [review]

**Pull request merged**
